### PR TITLE
[Snackbar] Update reference image for testSnackbarOverlayViewWithHighElevation

### LINKED ISSUE #9372: [Snackbar] Snapshot tests fail due to incorrect snapshot size
The `testSnackbarOverlayViewWithHighElevation` snapshot test fails because the reference image is an incorrect size.

failed_testSnackbarOverlayViewWithHighElevation_11_2@2x
reference_testSnackbarOverlayViewWithHighElevation_11_2@2x

### LINKED ISSUE #9380: [Snackbar] Snapshot test testSnackbarOverlayViewWithHighElevation is unreliable
As can be seen in https://github.com/material-components/material-components-ios/pull/9373, this test passes with different reference images.

### PR DESCRIPTION
**Pull request by bryanoltman**
Adds a runloop drain to `testSnackbarOverlayViewWithHighElevation` to reduce flakiness.

Fixes #9380

### CODE REVIEW

**Review by jverkoey — DISMISSED** [review]

File: components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m [hunk 1]
```diff
@@ -218,6 +218,7 @@ - (void)testSnackbarOverlayViewWithHighElevation {
   MDCSnackbarMessageView *messageView = [self snackbarMessageViewWithText:kItemTitleShort1Latin
                                                               actionTitle:kItemTitleShort2Latin];
   messageView.frame = CGRectMake(0, 0, kWidth, kHeightSingleLineText);
+  self.testManager.internalManager.overlayView.frame = CGRectMake(0, 0, 375, 667);
```
Comment (romoore) [inline components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m]: Tip: File an Issue to investigate why this is different on Xcode/Bazel builds and add a TODO here linking to that Issue.

**Review by romoore — DISMISSED** [review]

**Comment by romoore** [comment]
I'm worried that CI is passing with both the original (smaller) file and the new (larger) file without any change to the test. Your latest test code commit forced a specific size, but I suspect the test case may not be executing as we expect since it should not have passed for both the reference and new image files without that change in the test source.

**Comment by bryanoltman** [comment]
So it looks like this test was just a bit flaky. The last four commits demonstrate the original image was correct and that the failures/successes are reproducible. The failing commits (b1956ef and c857d3b) have the Xcode-generated snapshot, while the passing commits (25fcf47 and b2bcbee) have the kokoro-generated snapshot. I'm still not sure why this is the case, but this PR at least makes the test a bit less flaky.

File: components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m [hunk 2]
```diff
@@ -225,6 +225,12 @@ - (void)testSnackbarOverlayViewWithHighElevation {
                                                         animated:NO
                                                       completion:nil];
 
+  XCTestExpectation *expectation = [self expectationWithDescription:@"completed"];
```
Comment (romoore) [inline components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m:231]: Tip: Add a comment here explaining that there was some flakiness when running Bazel/Xcode versions of the test and that's why a runloop drain was added.

**Review by romoore — APPROVED** [review]

File: components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m (hunk 2 above)
Comment (bryanoltman) [reply components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m:231]: Done

File: components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m (hunk 1 above)
Comment (bryanoltman) [reply components/Snackbar/tests/snapshot/MDCSnackbarMessageViewSnapshotTests.m]: Done.

**Pull request merged**